CMakeLists.txt

```diff
@@ -147,6 +147,10 @@ if(WIN32)
 	list(APPEND CORE_VFS_SRC ${CMAKE_SOURCE_DIR}/src/util/vfs/vfs-fd.c ${CMAKE_SOURCE_DIR}/src/platform/windows/vfs-w32.c)
 	file(GLOB OS_SRC ${CMAKE_SOURCE_DIR}/src/platform/windows/*.c)
 	source_group("Windows-specific code" FILES ${OS_SRC})
+	if(MSVC)
+		include_directories(SYSTEM ${CMAKE_SOURCE_DIR}/src/platform/msvc)
```

Comment:
I suppose I can just put getopt in windows as "getopt_msvc" and add some ifdefs.